eng/pipelines/common/checkout-job.yml

```diff
@@ -32,11 +32,7 @@ jobs:
   displayName: Checkout
 
   pool:
-    ${{ if eq(variables['System.TeamProject'], 'public') }}:
-      name: Hosted MacOS
-
-    ${{ if ne(variables['System.TeamProject'], 'public') }}:
-      name: Hosted Mac Internal
+    name: Hosted MacOS
```

Comment:
If the people who know about the implications of doing this say it's the right thing, OK.  But generally we maintain resource isolation for the official builds

Reply:
I talked to them and we should be fine, and actually this way of using pools is deprecated, so I just updated it to specify a vmImage. Since this is the only hosted pool we use, the others don't  require any updates.